src/test/functional/features/relationship-date-formed.feature

```diff
@@ -30,6 +30,16 @@ Feature: Relationship date formed
     When I click "Continue"
     Then the page should include "You have entered a date that is in the future. Enter a date that is in the past before continuing."
 
+  Scenario: Entering date less than a year
+    Given I select "Day"
+    And I type "1"
+    Given I select "Month"
+    And I type "1"
+    Given I select "Year"
+    And I type "2021"
```

Comment:
Perhaps it might be better to make a new step that handles entering a dynamic date (e.g. 6 months ago). Rather than hard coding a date, since this will cause the test to failing next year.

Reply:
yh i was thinking that but then i thought, is this going to repeatly going to break